src/i18n/en.js

```diff
@@ -120,6 +120,26 @@ const messages_en = {
     title: 'Plans sized up for your business',
     until_x_subscribers: `Until {subscribers} Contacts.`,
   },
+  checkoutProcessForm: {
+    billing_information_title: `BILLING INFORMATION`,
```

Comment:
What do you think about using uppercase by styles?

```suggestion
    billing_information_title: `Billing Information`,
```

Reply:
Yes, I agree with it. I will change.

Reply:
Done